back/src/forms/pdf/helpers.ts

```diff
@@ -0,0 +1,97 @@
+import { PackagingInfo } from "@td/codegen-back";
+import { pluralize } from "@td/constants";
+import Decimal from "decimal.js";
+
+const packagingsLabel = {
+  FUT: "Fût",
+  GRV: "GRV",
+  CITERNE: "Citerne",
+  BENNE: "Benne",
+  PIPELINE: "Conditionné pour pipeline",
+  AUTRE: "Autre"
+};
+
+/**
+ * Permet de calculer les lignes du tableau de conditionnement <PackagingInfosTable />
+ */
+export function getPackagingsRows(packagingInfos: PackagingInfo[]) {
+  const packagingsByType: { [key: string]: PackagingInfo[] } =
+    packagingInfos.reduce((acc, packaging) => {
+      return {
+        ...acc,
+        [packaging.type]: [...(acc[packaging.type] ?? []), packaging]
+      };
+    }, {});
+
+  const packagingsRows = Object.keys(packagingsByType).map(type => {
+    const packagings = packagingsByType[type];
+
+    // Nombre total de conditionnement pour un type donné
+    const totalQuantity = packagings.reduce(
+      (totalQuantity, p) => totalQuantity + (p.quantity ?? 0),
+      0
+    );
+
+    let conditionnement: string = pluralize(
```

Comment:
Je sais qu'on utilise de plus en plus de français dans les commentaires, mais partout ailleurs on utilise le terme `packaging` dans le code. Il me semble que ca serait plus clair de garder ça
```suggestion
    let packagingString: string = pluralize(
```